shared/middleware/helpers/bucket_helper.rb

```diff
@@ -92,7 +92,7 @@ def get(encrypted_channel_id, filename, if_modified_since = nil, version = nil)
     end
     key = s3_path owner_id, channel_id, filename
     begin
-      s3_object = s3_get_object(key, if_modified_since, version)
+      s3_object = s3_get_object(key, if_modified_since.presence, version)
```

Comment:
you shouldn't use `presence` here because its part of active support, which we don't have in pegasus, or at least we don't get it without explicitly including it like we do in rails. these shared APIs are still used in pegasus by weblab shares on codeprojects.org, unfortunately.

Reply:
Good call, reverted ad8cbfbe34ecb24274459d893a02cbdf214e961e.